packages/web-components/fast-foundation/src/avatar/avatar.template.ts

```diff
@@ -0,0 +1,35 @@
+import { html, when } from "@microsoft/fast-element";
+import { Avatar } from "./avatar";
+
+/**
+ * The template for {@link @microsoft/fast-foundation#Avatar} component.
+ * @public
+ */
+export const AvatarTemplate = html<Avatar>`
+    <template>
+        <div
+            class="coin ${x => (x.shape === "square" ? "square" : "circle")}"
+            part="coin"
+            style="${x =>
+                x.fill || x.color
+                    ? `background-color: var(--avatar-fill-${x.fill}); color: var(--avatar-color-${x.color});`
+                    : void 0}"
+        >
+            <a class="link" part="link" href="${x => (x.link ? x.link : void 0)}">
+                ${when(
+                    x => x.imgSrc,
+                    html`
+                        <img
+                            src="${x => x.imgSrc}"
+                            alt="${x => x.alt}"
+                            tabindex="${x => (!x.link ? "0" : void 0)}"
```

Comment:
@litteredwitherrors can you elaborate on this being focusable when there is no link?

Reply:
Currently, adding a link adds an `href` to the anchor tag. The anchor is still present when `link` is omitted, and focus is still delegated to it. I am noticing that there is no focus indication, though, on any of the components. I'm going to have to fix this.

Reply:
@chrisdholt Apparently, `delegatesFocus` was never added. I added it and that fixed all the weird focus issues. I'll push the fix soon.

Reply:
@litteredwitherrors still seeing this here - I think if the anchor is a link it should receive focus now. If it's not, I'd expect it to not focusable. aShould be able to have this just removed.

Reply:
@chrisdholt Strange. As I'm testing it right now, focus does not fall on a component that does not have a `link` attribute.

Reply:
Is this the most recent instance of the code? You mentioned pulling this, but GH doesn't seem to recognize that.